doc/src/tutorial/001_adding_lalrpop.md

```diff
@@ -78,4 +78,17 @@ create a submodule `foo::grammar`):
 lalrpop_mod!(grammar);
 ```
 
+## Attributes
+
+The code that LALRPOP generates sometimes falls afoul of code linters for
+cosmetic reasons which don't apply to macro generated code. Currently, there is
+not a blanket `#[automatically_derived]` attribute in the generated parser
+because action code provided by the user may be worth linting on. Users can
+provide any number of attributes which they would like applied to the generated
+module. This is particularly useful for suppressing noisy lints.
+
+```rust
+lalrpop_mod!(#[allow(#[allow(clippy::ptr_arg)]#[rustfmt::skip] parser);
```

Comment:
Is this correct? the nesting and formatting is strange. I would expect something like:

```rust
lalrpop_mod!(
    #[allow( // missing something here??
    #[allow(clippy::ptr_arg)]
    #[rustfmt::skip]
    parser);
```

Reply:
Whoops

Reply:
@Pat-Lafon I would also include the newlines, because that's how I would write attributes on an normal Rust module and it's more readable IMHO. Of course, that's just my opinion, and you can keep it the current way.